server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -809,27 +814,40 @@ public void run() {
     // Periodically check that metadata of tablets matches what is held in memory
     ThreadPools.watchCriticalScheduledTask(ThreadPools.getServerThreadPools()
         .createGeneralScheduledExecutorService(aconf).scheduleWithFixedDelay(() -> {
-          final SortedMap<KeyExtent,Tablet> onlineTabletsSnapshot = onlineTablets.snapshot();
-
-          Map<KeyExtent,Long> updateCounts = new HashMap<>();
+          Instant start = Instant.now();
```

Comment:
This lambda is starting to get long, wonder if it should be pulled out to a function.

Reply:
I agree. It is very hard to read and decipher. I think the two ThreadPools methods could be consolidated.